Documentation.app/Contents/MacOS/Generation examples.playground/Pages/Limitations.xcplaygroundpage/Contents.swift

```diff
@@ -7,4 +7,15 @@
 /*:
  # Limitations
  
+ Bow OpenAPI is very powerful and generates a ready to use network client in Swift that is as good as the specification you provide. If the specification is poor or flaky, the generated code will have the same problems. Generating code according to this specification is a way of honoring it, but it is important that this contract is also honored by the backend side.
+ 
+ Nevertheless, Bow OpenAPI has some other limitations.
+ 
+ ## Inline data types
```

Comment:
I think we should add information about `encoding` for body parameters - it is always being done with JSON encoder.